src/api.h

```diff
@@ -320,6 +348,7 @@ AWS_CRT_API void aws_crt_signing_config_aws_set_signed_body_header_type(
 AWS_CRT_API void aws_crt_signing_config_aws_set_expiration_in_seconds(
     aws_crt_signing_config_aws *signing_config,
     uint64_t expiration_in_seconds);
+AWS_CRT_API void aws_crt_signing_config_aws_set_date(aws_crt_signing_config_aws *signing_config, uint64_t timestamp);
```

Comment:
Param documentation in a separate later pass I assume?  Maybe at least put units into timestamp or meaning (nanos_since_epoch etc...)

Reply:
Will do.